ccd-definition/CaseEventToFields/CareSupervision/manageRepresentatives.json

```diff
@@ -3,9 +3,18 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "manageRepresentatives",
-    "CaseFieldID": "respondents_label",
+    "CaseFieldID": "warning_label",
     "PageID": 1,
     "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "manageRepresentatives",
+    "CaseFieldID": "respondents_label",
+    "PageID": 1,
+    "PageFieldDisplayOrder": 2,
     "DisplayContext": "READONLY",
     "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/manage-representatives/mid-event"
```

Comment:
I think the callbackUrl should be added in the first field. Could you please check if it is working?